src/library/mixxxlibraryfeature.cpp

```diff
@@ -176,12 +179,16 @@ void MixxxLibraryFeature::searchAndActivate(const QString& query) {
     activate();
 }
 
-#ifdef __ENGINEPRIME__
 void MixxxLibraryFeature::bindSidebarWidget(WLibrarySidebar* pSidebarWidget) {
-    // store the sidebar widget pointer for later use in onRightClick
+    // store the sidebar widget pointer for later use in onRightClick and
+    // updating track count
     m_pSidebarWidget = pSidebarWidget;
 }
-#endif
+
+void MixxxLibraryFeature::slotUpdateTrackCount() {
+    m_trackCount = m_pLibraryTableModel->rowCount();
+    m_pSidebarWidget->repaint();
```

Comment:
I wonder if we can make SidebarModel repaint the Tracks item only, ie. somehow invoke dataChanged([root item row ..])
What about
```
    const QString newTitle = title();
    m_pSidebarModel->getRootItem()->setLabel(newTitle);
```

Reply:
Never mind, that requires more calls.
The current solution is fine.